<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2530</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>2530</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>2488</width>
             <height>1075</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QTextEdit" name="textEdit"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>2488</width>
             <height>1075</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QTextEdit" name="textEdit_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2530</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File...</string>
    </property>
    <widget class="QMenu" name="menuChange_coding">
     <property name="title">
      <string>Change coding</string>
     </property>
     <addaction name="actionUTF_8"/>
     <addaction name="actionMacintosh"/>
     <addaction name="actionWindows1251"/>
     <addaction name="actionKOI_8"/>
    </widget>
    <addaction name="actionNew_file"/>
    <addaction name="actionOpen_file"/>
    <addaction name="actionSave_file"/>
    <addaction name="actionSave_file_as"/>
    <addaction name="menuChange_coding"/>
    <addaction name="actionOpen_in_new"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionSelect_all"/>
    <addaction name="actionFont_size"/>
    <addaction name="actionFont_style"/>
    <addaction name="actionFont_colour"/>
   </widget>
   <widget class="QMenu" name="menuCompile">
    <property name="title">
     <string>Compile</string>
    </property>
    <addaction name="actionSystem"/>
    <addaction name="actionQProcess"/>
   </widget>
   <widget class="QMenu" name="menuDesign">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionBackground_colour"/>
    <addaction name="actionTextBgColour"/>
    <addaction name="actionSynHL"/>
    <addaction name="actionSelection_colour"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCompile"/>
   <addaction name="menuDesign"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_file">
   <property name="text">
    <string>New file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file...</string>
   </property>
  </action>
  <action name="actionSave_file">
   <property name="text">
    <string>Save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionSystem">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>System</string>
   </property>
  </action>
  <action name="actionQProcess">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>QProcess</string>
   </property>
  </action>
  <action name="actionBackground_colour">
   <property name="text">
    <string>Background colour</string>
   </property>
  </action>
  <action name="actionTextBgColour">
   <property name="text">
    <string>Text background colour</string>
   </property>
  </action>
  <action name="actionSynHL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Syntax highlighting</string>
   </property>
  </action>
  <action name="actionFont_size">
   <property name="text">
    <string>Font size</string>
   </property>
  </action>
  <action name="actionFont_style">
   <property name="text">
    <string>Font style</string>
   </property>
  </action>
  <action name="actionFont_colour">
   <property name="text">
    <string>Font colour</string>
   </property>
  </action>
  <action name="actionSelection_colour">
   <property name="text">
    <string>Selection colour</string>
   </property>
  </action>
  <action name="actionSave_file_as">
   <property name="text">
    <string>Save file as...</string>
   </property>
  </action>
  <action name="actionUTF_8">
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="actionMacintosh">
   <property name="text">
    <string>Macintosh</string>
   </property>
  </action>
  <action name="actionWindows1251">
   <property name="text">
    <string>Windows1251</string>
   </property>
  </action>
  <action name="actionKOI_8">
   <property name="text">
    <string>KOI-8</string>
   </property>
  </action>
  <action name="actionOpen_in_new">
   <property name="text">
    <string>Open file in new page...</string>
   </property>
  </action>
  <action name="actionFont_colour_2">
   <property name="text">
    <string>Font colour</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
